clang/test/dpct/cublaslt.cu

```diff
@@ -19,13 +19,15 @@ void foo1 () {
   // CHECK-NEXT: uint64_t rows;
   // CHECK-NEXT: uint64_t cols;
   // CHECK-NEXT: int64_t ld;
+  // CHECK-NEXT: uint64_t batch_count;
   // CHECK-NEXT: matLayout = new dpct::blas_gemm::experimental::matrix_layout_t(type, rows, cols, ld);
   cublasLtMatrixLayout_t matLayout;
   cudaDataType type;
   uint64_t rows;
   uint64_t cols;
   int64_t ld;
-  cublasLtMatrixLayoutCreate(&matLayout, type, rows, cols, ld);
+  uint64_t batch_count;
+  cublasLtMatrixLayoutCreate(&matLayout, type, rows, cols, ld, batch_count);
```

Comment:
```suggestion
  cublasLtMatrixLayoutCreate(&matLayout, type, rows, cols, ld);
```
`cublasLtMatrixLayoutCreate` does not accept batch_count as parameter. https://docs.nvidia.com/cuda/cublas/#cublasltmatrixlayoutcreate

Reply:
Ok